src/main/java/duke/Event.java

```diff
@@ -0,0 +1,51 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+//@@author chengda300
+//Reused from https://nus-cs2103-ay2223s1.github.io/website/schedule/week2/project.html
+// with minor modifications
+
+/**
+ * Represents an event. An event has a description and a time that it will happen.
+ */
+public class Event extends Task {
+    private String at;
```

Comment:
Will it be better to give this a slightly more descriptive name?

Reply:
Yes it will be, it should be changed to event time I guess.